### DISCUSSION TITLE
I can't find the location of some files to translate

### DISCUSSION THREAD

**Discussion by Mozitux** [mediacms-io/mediacms#906] [Q&A]
Hello;

I am trying to fully translate MediaCMS into French and then make a pull request.

The modifications are progressing quite well, but I can't find the files to modify to translate the registration/connection parts of the CMS (like the placeholders).

![placeholder](https://github.com/mediacms-io/mediacms/assets/19860756/ecbc67c7-b54f-4063-a604-bd469fd3cfeb)

I tried to modify the 'registerPage.html' and 'signinPage.html' files located in /frontend/config/templates/static/, it does not modify the registration and connection pages.

**Comment by mgogoulos** [comment]
Hi @Mozitux , great to hear that you are working on this and also plan to open a PR. I have also been working on the translation and my work is on a branch (just pushed it / https://github.com/mediacms-io/mediacms/compare/main...feat-translations-all) . This is incomplete work unfortunately but very close to being completed. The idea is that all stringes get to be translated and served through the backend (Django), even the frontend ones. 

This work allows for adding translations and languages. Is your work replacing strings from English to French, or it allows the addition of other languages as well? 

There is a serious need to add dev documentation on how to develop locally, and there is another branch with this but it is also incomplete! 
Due to the way the app is setup, there are pages in frontend (like the ones you mention here) that are not used at all. What is used there are the Django templates, and  not the templates in frontend/ 

I believe that you can edit the fields (labels, placeholders etc) on users/forms.SignupForm . templates/account/signup.html is using this form, as this one is a setting provided by django-allauth in settings.py (`ACCOUNT_SIGNUP_FORM_CLASS = "users.forms.SignupForm"`) . So i guess this is the place to edit. 

Feel free to send me an invitation for a call to discuss more!

**Comment by mgogoulos** [comment]
They are now included in the latest release! Check it out on https://demo.mediacms.io/ and update!!!

Also if you want to help improving them, checkout https://github.com/mediacms-io/mediacms/blob/main/docs/admins_docs.md#20-translations
